plugins/woo-ai/includes/class-woo-ai-settings.php

```diff
@@ -0,0 +1,217 @@
+<?php
+/**
+ * Woo AI plugin main class
+ *
+ * @package Woo_AI
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+use Automattic\Jetpack\Constants;
+
+/**
+ * Woo_AI Main Class.
+ */
+class Woo_AI_Settings {
+
+	/**
+	 * Plugin instance.
+	 *
+	 * @var Woo_AI_Settings
+	 */
+	protected static $instance = null;
+
+
+	/**
+	 * Settings ID.
+	 *
+	 * @var id
+	 */
+	protected $id = 'woo-ai-settings-tab';
+
+	/**
+	 * Main Instance.
+	 */
+	public static function instance() {
+		self::$instance = is_null( self::$instance ) ? new self() : self::$instance;
+		return self::$instance;
+	}
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		add_action( 'admin_enqueue_scripts', array( $this, 'add_woo_ai_settings_script' ) );
+		add_action( 'woocommerce_settings_save_advanced', array( $this, 'action_save_woo_ai_settings_tab' ) );
+		add_action( 'woocommerce_settings_page_init', array( $this, 'add_ui' ) );
+
+		$this->add_sanitization_hooks();
+	}
+
+	/**
+	 * Add UI related hooks.
+	 */
+	public function add_ui() {
+		add_filter( 'woocommerce_get_settings_advanced', array( $this, 'add_woo_ai_settings' ), 10, 2 );
+	}
+
+	/**
+	 * Save settings.
+	 */
+	public function action_save_woo_ai_settings_tab() {
+		WC_Admin_Settings::save_fields(
+			$this->get_woo_ai_settings()
+		);
+	}
+
+	/**
+	 * Add sanitization hooks.
+	 */
+	public function add_sanitization_hooks() {
+		$settings = $this->get_woo_ai_settings();
+
+		foreach ( $settings as $setting ) {
+			if ( in_array( $setting['type'], array( 'text', 'textarea' ), true ) ) {
+				add_filter( 'woocommerce_admin_settings_sanitize_option_' . $setting['id'], array( $this, 'strip_tags_field_value' ) );
+			}
+		}
+	}
+
+	/**
+	 * Sanitize field value.
+	 *
+	 * @param string $raw_value The current section.
+	 */
+	public function strip_tags_field_value( $raw_value ) {
+		return wp_strip_all_tags( $raw_value ?? '' );
+	}
+
+	/**
+	 * Add settings to the AI section.
+	 *
+	 * @param array  $settings The original settings array.
+	 * @param string $current_section The current section.
+	 */
+	public function add_woo_ai_settings( $settings = array(), $current_section = null ) {
+		if ( 'features' !== $current_section ) {
+			return $settings;
+		}
+
+		return array_merge(
+			$this->get_woo_ai_settings(),
+			$settings
+		);
```

Comment:
A bit nitpicky but I personally think this is easier to follow and read:
```suggestion
		if ( 'features' === $current_section ) {
			$settings = array_merge(
			$this->get_woo_ai_settings(),
			$settings
		);
		}

		return $settings;
```

Reply:
Works for me 👍🏻